### ISSUE TITLE
Finalize MIC Biasing

### ISSUE THREAD

**Issue by softerhardware** [softerhardware/CWKeyer#5]
The mic biasing used is based on the reform laptop:
https://www.crowdsupply.com/mnt/reform

The bias is applied to the tip and not the ring. This may not work with all microphones.

Test with various microphones and change or update the bias so that it works with as wide a range of microphones as easily possible.

**Comment by dl1ycf** [comment]
Well, depends what you want. If you provide "mic and bias" both on the tip, you have the ring free for PTT. If you provide mic and bias on on tip+ring you have to use two plugs for the microphone (the second one going to the PTT jack). So at least via jumpering, it should be possible to have Mic+Bias tied  together on one contact (ring or tip) and PTT on the other.

P.S.: would be nice if one could remove the bias via jumpers. No problem for me since I have a decoupling cap, but I use a dynamical mic with built-in battery-driven preamp  and this would not like the bias if there were no decoupler.

Bottom line: if  you want to be everybody's Darling, perhaps the mic connector must have a graveyard of jumpers.
